apex/contrib/test/group_norm/test_group_norm.py

```diff
@@ -202,7 +203,12 @@ def test_group_norm_inductor(self):
             y.backward(dy)
 
         from torch._dynamo.utils import counters
-        self.assertNotIn('graph_break', counters, "Shouldn't see any graph breaks.")
+        # TODO: Remove this when 3.9 is no longer supported
+        if sys.version_info < (3, 10):
+            num_graph_breaks = sum(counters["graph_break"].values())
+        else:
+            num_graph_breaks = counters["graph_break"].total()
```

Comment:
Is `counters["graph_break"]` guaranteed to exist? Would it be necessary to check its existence like so:
```Python
...
        num_graph_breaks = 0
        if "graph_break" in counters:
            if sys.version_info < (3, 10):
                num_graph_breaks = sum(counters["graph_break"].values())
            else:
                num_graph_breaks = counters["graph_break"].total()
...
```

Reply:
Okay, sounds good then!